cpp/include/cudf/io/parquet.hpp

```diff
@@ -525,6 +533,11 @@ class parquet_writer_options {
   {
     _column_chunks_file_path.assign(file_path);
   }
+
+  /**
+   * @brief Sets a pointer to the decimal precision vector.
+   */
+  void set_decimal_precisions(std::vector<uint8_t> const& dp) { _decimal_precisions = dp; }
```

Comment:
To avoid a copy when input is an r-value:
```suggestion
  void set_decimal_precisions(std::vector<uint8_t> dp) { _decimal_precisions = std::move(dp); }
```

Reply:
Thanks, I changed that and also updated the comments on the set and get to indicate they are no longer pointers.